changelog.mdx

```diff
@@ -5,6 +5,15 @@ description: Updates to Chalk!
 
 ---
 
+## July 26, 2024
+
+### Dashboard improvements
+
+* The offline query page of the dashboard now shows live query progress. After query completion, the query page will
+  also show how long each resolver took to run."
+* The Kubernetes resource page in the dashboard now allows you to group resources by application, component, and other
+  common groupings.
```

Comment:
very open to edits

Reply:
It might be nice to add something about it showing which nodes are running in the cluster, like "it also now shows which kinds of hardware resources are currently running".